website/playground/Playground.js

```diff
@@ -154,6 +131,7 @@ class Playground extends React.Component {
   render() {
     const { worker, version } = this.props;
     const { content, options } = this.state;
+    const handleOptionValueChange = this.handleOptionValueChange.bind(this);
```

Comment:
This shouldn't be done in `render`.

Reply:
Both aren't recommended. This creates a new function value on every render, which triggers re-rendering of components this function passed to.

Reply:
I know, but arrow functions used in other part of this `render`, I don't think it cost much.